docker-compose.yml

```diff
@@ -439,6 +439,7 @@ services:
       - "${PAYMENT_SERVICE_PORT}"
     environment:
       - PAYMENT_SERVICE_PORT
+      - PAYMENT_SERVICE_BUGSNAG_API_KEY
```

Comment:
Based on how the Terraform configuration looks, it seems that we need to keep the `BUGSNAG_API_KEY` in the app and add the mapping in here, so what should work is:
```
- BUGSNAG_API_KEY=${PAYMENT_SERVICE_BUGSNAG_API_KEY}
```
So in the `.env` file we will have `PAYMENT_SERVICE_BUGSNAG_API_KEY` and in the code in the app `BUGSNAG_API_KEY`.

Please let me know if it works.

Reply:
I made the change and it works, I could still see the error in Bugnsag